plugins/serializers/influx/influx_test.go

```diff
@@ -275,6 +275,22 @@ var tests = []struct {
 		),
 		output: []byte("cpu abc=123i 1519194109000000042\ncpu def=456i 1519194109000000042\n"),
 	},
+	{
+		name:     "split_fields_overflow",
```

Comment:
This test passes without the change to the serializer, is it possible to modify it to require the change?

Reply:
The test now fails if the change is not implemented